<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullKeyboard</class>
 <widget class="QWidget" name="FullKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
border: 2px solid #Fb8800;
background-color: #FFFFFF;
}

QPushButton{
border: 2px solid #Fb8800;
font-family: &quot;Microsoft YaHei&quot;;
font-weight:bold;
font-size:15px;
color: #303030;
}

QPushButton:pressed {
    background-color:rgb(253,196,128);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>740</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn1{
background-color: #FFFFFF;
background-image: url(:/icons/btnS1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn2{
background-color: #FFFFFF;
background-image: url(:/icons/btnS2.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn3{
background-color: #FFFFFF;
background-image: url(:/icons/btnS3.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn6{
background-color: #FFFFFF;
background-image: url(:/icons/btnS6.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn4{
background-color: #FFFFFF;
background-image: url(:/icons/btnS4.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn5{
background-color: #FFFFFF;
background-image: url(:/icons/btnS5.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn7{
background-color: #FFFFFF;
background-image: url(:/icons/btnS7.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn8{
background-color: #FFFFFF;
background-image: url(:/icons/btnS8.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn9{
background-color: #FFFFFF;
background-image: url(:/icons/btnS9.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn0{
background-color: #FFFFFF;
background-image: url(:/icons/btnS0.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEqual">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnEqual{
background-color: #FFFFFF;
background-image: url(:/icons/btnSEqual.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>124</y>
      <width>65</width>
      <height>104</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnCancel{
background-color: #FF0000;
color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShift">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>232</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnShift{
background-color: #B3B3B3;
background-image: url(:/icons/caseSwitch.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRBracket">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnRBracket{
background-color: #FFFFFF;
background-image: url(:/icons/btnSRBracket.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQ">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnW">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>w</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnR">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnE">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnT">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnY">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnU">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>u</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnI">
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnO">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnP">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClr">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnLeft{
background-color: #B3B3B3;
}</string>
    </property>
    <property name="text">
     <string>clr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>658</x>
      <y>70</y>
      <width>92</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnDelete{
background-color: #B3B3B3;
background-image: url(:/icons/btnDel.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHyphen">
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>124</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnHyphen{
background-color: #FFFFFF;
background-image: url(:/icons/btnSHyphen.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>124</y>
      <width>65</width>
      <height>104</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnOk{
background-color: #00BD00;
color:#FFFFFF;
QPushButton:!enable{
background-color: #CCCCCC;
background: transparent;
}
}
</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnA">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnS">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnD">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>d</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnF">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnG">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnH">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJ">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>j</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnK">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnL">
    <property name="geometry">
     <rect>
      <x>522</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>l</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLSquBra">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnLSquBra{
background-color: #FFFFFF;
background-image: url(:/icons/btnSLSquBra.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZ">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRSquBra">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>178</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnRSquBra{
background-color: #FFFFFF;
background-image: url(:/icons/btnSRSquBra.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnX">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnC">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnV">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnB">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnN">
    <property name="geometry">
     <rect>
      <x>374</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnM">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnComma">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnComma{
background-color: #FFFFFF;
background-image: url(:/icons/btnSComma.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPoint">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnPoint{
background-color: #FFFFFF;
background-image: url(:/icons/btnSPoint.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLBracket">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnLBracket{
background-color: #FFFFFF;
background-image: url(:/icons/btnSLBracket.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuote">
    <property name="geometry">
     <rect>
      <x>698</x>
      <y>232</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnQuote{
background-color: #FFFFFF;
background-image: url(:/icons/btnSQuote.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSwitch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>286</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnSwitch{
background-color: #B3B3B3;
background-image: url(:/icons/btnSwitch.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>286</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnRight{
background-color: #B3B3B3;
}</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeft">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>286</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnLeft{
background-color: #B3B3B3;
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSpace">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>286</y>
      <width>546</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnSpace{
background-image: url(:/icons/btnLogo.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
